2018/05/26 22:32:11 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /_cat/indices?pretty HTTP/1.1", upstream: "http://172.18.0.2:9200/_cat/indices?pretty", host: "localhost:9200"
2018/05/26 22:32:16 [error] 7#7: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /_cat/indices?pretty HTTP/1.1", upstream: "http://172.18.0.2:9200/_cat/indices?pretty", host: "localhost:9200"
2018/05/26 22:32:21 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /_cat/indices?pretty HTTP/1.1", upstream: "http://172.18.0.2:9200/_cat/indices?pretty", host: "localhost:9200"
2018/05/27 06:07:24 [error] 7#7: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.18.0.3:9200/", host: "localhost:9200"
2018/05/27 06:25:43 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /elasticsearch/_msearch HTTP/1.1", upstream: "http://172.18.0.5:5601/elasticsearch/_msearch", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:25:43 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/metrics/vis/data HTTP/1.1", upstream: "http://172.18.0.5:5601/api/metrics/vis/data", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:25:48 [error] 7#7: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /elasticsearch/_msearch HTTP/1.1", upstream: "http://172.18.0.5:5601/elasticsearch/_msearch", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:25:48 [error] 7#7: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/metrics/vis/data HTTP/1.1", upstream: "http://172.18.0.5:5601/api/metrics/vis/data", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:25:53 [error] 7#7: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /elasticsearch/_msearch HTTP/1.1", upstream: "http://172.18.0.5:5601/elasticsearch/_msearch", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:25:53 [error] 7#7: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/metrics/vis/data HTTP/1.1", upstream: "http://172.18.0.5:5601/api/metrics/vis/data", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:25:58 [error] 7#7: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /elasticsearch/_msearch HTTP/1.1", upstream: "http://172.18.0.5:5601/elasticsearch/_msearch", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:25:58 [error] 7#7: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/metrics/vis/data HTTP/1.1", upstream: "http://172.18.0.5:5601/api/metrics/vis/data", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:26:03 [error] 7#7: *12 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /elasticsearch/_msearch HTTP/1.1", upstream: "http://172.18.0.5:5601/elasticsearch/_msearch", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:26:03 [error] 7#7: *12 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/metrics/vis/data HTTP/1.1", upstream: "http://172.18.0.5:5601/api/metrics/vis/data", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:26:08 [error] 7#7: *12 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /elasticsearch/_msearch HTTP/1.1", upstream: "http://172.18.0.5:5601/elasticsearch/_msearch", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:26:08 [error] 7#7: *16 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/metrics/vis/data HTTP/1.1", upstream: "http://172.18.0.5:5601/api/metrics/vis/data", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:33:21 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/console/proxy?path=_mapping&method=GET HTTP/1.1", upstream: "http://172.18.0.4:5601/api/console/proxy?path=_mapping&method=GET", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:33:21 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/console/proxy?path=_aliases&method=GET HTTP/1.1", upstream: "http://172.18.0.4:5601/api/console/proxy?path=_aliases&method=GET", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:33:26 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/console/proxy?path=_mapping&method=GET HTTP/1.1", upstream: "http://172.18.0.4:5601/api/console/proxy?path=_mapping&method=GET", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:33:26 [error] 7#7: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/console/proxy?path=_aliases&method=GET HTTP/1.1", upstream: "http://172.18.0.4:5601/api/console/proxy?path=_aliases&method=GET", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:41:09 [error] 7#7: *85 upstream prematurely closed connection while reading response header from upstream, client: 172.18.0.1, server: , request: "GET /v1/city/count HTTP/1.1", upstream: "http://172.18.0.5:8080/v1/city/count", host: "localhost", referrer: "http://localhost/"
2018/05/27 06:50:01 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /elasticsearch/_msearch HTTP/1.1", upstream: "http://172.18.0.6:5601/elasticsearch/_msearch", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:50:14 [error] 7#7: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /elasticsearch/_msearch HTTP/1.1", upstream: "http://172.18.0.6:5601/elasticsearch/_msearch", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:50:27 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /elasticsearch/_msearch HTTP/1.1", upstream: "http://172.18.0.6:5601/elasticsearch/_msearch", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:50:46 [error] 7#7: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /elasticsearch/_msearch HTTP/1.1", upstream: "http://172.18.0.6:5601/elasticsearch/_msearch", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 06:51:08 [error] 7#7: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /match?bidder_id=4&external_user_id=RX-89efe22f-20a0-4c7a-82cb-86f0b72b320e&rndcb=8453343383 HTTP/1.1", upstream: "http://172.18.0.7:8080/match?bidder_id=4&external_user_id=RX-89efe22f-20a0-4c7a-82cb-86f0b72b320e&rndcb=8453343383", host: "ads.kiosked.com", referrer: "http://www.pythonforbeginners.com/dictionary/how-to-use-dictionaries-in-python"
2018/05/27 06:51:15 [error] 7#7: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /usnci?i=1&ec=rhy&r=http%3A%2F%2Fsync.1rx.io%2Fusersync%2Ftlvmedia%2F&rndcb=8865138840 HTTP/1.1", upstream: "http://172.18.0.7:8080/usnci?i=1&ec=rhy&r=http%3A%2F%2Fsync.1rx.io%2Fusersync%2Ftlvmedia%2F&rndcb=8865138840", host: "cs.ffbtas.com", referrer: "http://www.pythonforbeginners.com/dictionary/how-to-use-dictionaries-in-python"
2018/05/27 06:51:19 [error] 7#7: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /elasticsearch/_msearch HTTP/1.1", upstream: "http://172.18.0.6:5601/elasticsearch/_msearch", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 07:05:15 [emerg] 1#1: host not found in upstream "elasticsearch" in /etc/nginx/conf.d/default.conf:32
2018/05/27 08:04:34 [error] 10#10: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/console/proxy?path=_mapping&method=GET HTTP/1.1", upstream: "http://172.18.0.4:5601/api/console/proxy?path=_mapping&method=GET", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 08:04:35 [error] 10#10: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/console/proxy?path=_aliases&method=GET HTTP/1.1", upstream: "http://172.18.0.4:5601/api/console/proxy?path=_aliases&method=GET", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 08:24:53 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/console/proxy?path=_mapping&method=GET HTTP/1.1", upstream: "http://172.18.0.5:5601/api/console/proxy?path=_mapping&method=GET", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
2018/05/27 08:24:54 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "POST /api/console/proxy?path=_aliases&method=GET HTTP/1.1", upstream: "http://172.18.0.5:5601/api/console/proxy?path=_aliases&method=GET", host: "localhost:5601", referrer: "http://localhost:5601/app/kibana"
